services/order.py

```diff
@@ -0,0 +1,33 @@
+from db.models import Order, Ticket
+from django.db import transaction
+from django.contrib.auth import get_user_model
+
+
+@transaction.atomic
+def create_order(tickets: list[dict],
+                 username: str,
+                 date: str = None) -> None:
```

Comment:
Code Style Issue: It's better to use double quotes for strings in Python, as per PEP 8 style guide.